apache-rat-tasks/pom.xml

```diff
@@ -118,29 +124,12 @@
             <configuration>
               <includeArtifactIds>apache-rat-core</includeArtifactIds>
               <excludeClassifiers>tests</excludeClassifiers>
-              <include>META-INF/RELEASE_NOTES.txt</include>
+              <includes>META-INF/RELEASE_NOTES.txt</includes>
               <outputDirectory>${project.build.outputDirectory}</outputDirectory>
             </configuration>
           </execution>
         </executions>
       </plugin>
-      <plugin>
```

Comment:
We should check if the project still works in Eclipse. I remember that Eclipse sometimes does strange things with generated files that are copied into the source tree ....

Reply:
Filed RAT-474 to handle Eclipse problems separately (they happen on master and this feature branch)!